podcasts/Sharing/SharingModal.swift

```diff
@@ -128,28 +129,14 @@ enum SharingModal {
 
         let hostingController = ThemedHostingController(rootView: modalView, theme: Theme(previewTheme: .contrastLight))
         viewController.present(hostingController, animated: true)
-
-        Analytics.track(.podcastShared, source: source, properties: ["type": option.analyticsType])
     }
 }
 
 extension SharingModal.Option {
-    fileprivate var analyticsType: String {
```

Comment:
This variable was not being used anywhere and it's value are outdated so I removed it from the codebase.
The value is calculated more dynamically on `ShareDestination.type` method